CRM/Civicase/Hook/BuildForm/RemoveExportActionFromReports.php

```diff
@@ -67,7 +67,7 @@ private function removeExportOption(array &$options) {
    *   returns a boolean to determine if hook will run or not.
    */
   private function shouldRun(CRM_Core_Form $form) {
-    return !CRM_Core_Permission::check(CRM_Civicase_Service_ExportPermission::NAME)
+    return !CRM_Core_Permission::check(CRM_Civicase_Hook_Permissions_ExportCasesAndReports::PERMISSION_NAME)
```

Comment:
Can we do use CRM_Civicase_Hook_Permissions_ExportCasesAndReports as ExportCasesAndReports at the top of this file so we don't have this line as long